src/libmongoc/doc/mongoc_server_api_version_t.rst

```diff
@@ -0,0 +1,34 @@
+:man_page: mongoc_server_api_version_t
+
+mongoc_server_api_version_t
+===========================
+
+A representation of server API version numbers.
+
+Synopsis
+--------
+
+Used to specify which version of the MongoDB server's API to use for driver connections.
+
+Supported API Versions
+----------------------
+
+The driver currently supports the following MongoDB API versions:
+
+====================  ======================
+Enum value            MongoDB version number
+====================  ======================
+MONGOC_SERVER_API_V1  1
+====================  ======================
```

Comment:
Referring to version string instead of version number and quoting the value would help convey that versions are not limited to numbers but can be any arbitrary string (e.g. "luscious-lollipop"):

```suggestion
====================  ======================
Enum value            MongoDB version string
====================  ======================
MONGOC_SERVER_API_V1  "1"
====================  ======================
```